devops/rpmbuild/extrafiles/basecoin/usr/share/basecoin/key2.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
correct me if I am wrong, but the key file must be generated on a node. or this is like an example for tutorials? I am not getting it